src/bluesearch/entrypoint/database/topic_extract.py

```diff
@@ -160,7 +161,10 @@ def run(
     elif article_source is ArticleSource.PUBMED:
         for path in inputs:
             logger.info(f"Processing {path}")
-            articles = ElementTree.parse(input_path)
+            with gzip.open(path) as mygzip:
+                articles_str = mygzip.read()
+            articles = ElementTree.fromstring(articles_str)
+
```

Comment:
So after this we are assuming in `topic-extract` that the input is a `.xml.gz` file when the source is `PubMed`.
But in `parse` don't we assume we have the `xml` file (i.e. that the `.gz` was unzziped)?
https://github.com/BlueBrain/Search/blob/6e362760ef0d2bbb6cd583dfb363313d222737e2/src/bluesearch/entrypoint/database/parse.py#L130-L136

Maybe this is more a question for @jankrepl who opened #551.

Reply:
I think that as soon as we can do without unzipping and use directly the raw files, it is better (less intermediate/manual) steps. If there are other inconsistencies, we should maybe address them. Do you want me to make the changes in this in the current PR?

Reply:
I am fine leaving this part outside of the current PR, but maybe just open an issue with a reference to this PR so we can address it next :)